gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemIntegrationTest.java

```diff
@@ -1474,7 +1477,7 @@ private void testGlobStatusFlatConcurrent(GlobAlgorithm globAlgorithm) throws IO
     ghfs.mkdirs(new Path("/directory1/subdirectory1"));
     createFile(new Path("/directory1/subdirectory1/file1"), "data".getBytes(UTF_8));
 
-    FileStatus[] rootDirStatuses = ghfs.globStatus(new Path("/d*"));
+    FileStatus[] rootDirStatuses = ghfs.globStatus(new Path("/directory1*"));
```

Comment:
Why do we need to change pattern?

Reply:
Out of scope. yeah. Will do the cleanup.